.vitepress/theme/components/Banner.vue

```diff
@@ -22,16 +22,30 @@ function dismiss() {
 
 <template>
   <div class="banner" v-if="open">
-    <a traget="_blank"></a>
+    <p class="vt-banner-text">
+      <span class="vt-text-primary">VueConf Toronto</span>
+      <span class="vt-tagline"> - Join the premier Vue.js conference</span>
+      | 9-10 Nov 2023 <span class="vt-place"> - Toronto, Canada</span>
+      <a
+        target="_blank"
+        class="vt-primary-action"
+        href="https://vuetoronto.com?utm_source=vuejs&utm_content=top_banner"
+        >Register <span class="vt-time-now">Now</span></a
+      >
+    </p>
     <button @click="dismiss">
       <VTIconPlus class="close" />
     </button>
+    <p class="vt-banner-text vt-coupon">
+      <span class="vt-text-primary">Use code</span> VUEJS
+      <span class="vt-text-primary">to get 15% off</span>
+    </p>
```

Comment:
¿Esto no se traduce?

Reply:
Anteriormente no se han traducido los banners, solo los ads dentro de la documentación.

Reply:
Y no se podría considerar, para el engagement?

Reply:
translated it 🙌🏼